checker/jtreg/nullness/onlyannotatedfor/AnnotatedForWithUse.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * @test
+ *
+ * @summary Test that difference default applies to unannotated code.
+ * @compile/fail/ref=OnlyAnnotatedForWithUse.out -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AonlyAnnotatedFor AnnotatedForWithUse.java
```

Comment:
It's a good idea to use the file name of the test driver as a prefix rather than as a suffix of the expected output file - that puts the files next to each other and makes the grouping clear.